### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
The aim of this project was to be able to deal with UX design, visual hierarchy, and typography which we have applied in this project. For the positioning, we have used float, flex, and grid. Feedbacks are highly accepted. Thanks

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+.article-flex {
+  display: flex;
+  flex-direction: row;
+  margin: 20px auto;
+  padding: 10px;
+}
+
+.column1 {
+  /* text-align: center; */
```
Comment (andresporras3423) [inline style.css]: remove commented code

File: style.css [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+.date-title {
+  font-size: medium;
+  color: #fff;
+  padding-top: 30px;
+}
+
+/* .date-title p{
+
+} */
```
Comment (andresporras3423) [inline style.css]: Remove commented code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+            <article class="article">
+                <p>Lorem Ipsum Dolor</p>
+            </article>
+            <article class="article">
+                <p>Lorem Ipsum Dolor</p>
+            </article>
+          </div>
+        </div>
+        <div class="article-flex">
+            <div class="column1">
```
Comment (andresporras3423) [inline index.html]: Here use article tag instead of div

File: index.html [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+                    Can Data Visualization Improve The Mobile Web Experience?
+                </h2>
+                <p>
+                    by Sansa Stark Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Accusantium voluptates amet rem illum
+                    sequi illo, ex quis eum error ab totam molestias! Pariatur,
+                    qui harum! Ad molestias nulla quisquam unde.
+                </p>
+            </div>
+            <div class="column1">
```
Comment (andresporras3423) [inline index.html]: Use the article tag instead of div

File: index.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+                </h2>
+                <p>
+                    by Sansa Stark Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Accusantium voluptates amet rem illum
+                    sequi illo, ex quis eum error ab totam molestias! Pariatur,
+                    qui harum! Ad molestias nulla quisquam unde.
+                </p>
+            </div>
+
+            <div class="column1">
```
Comment (andresporras3423) [inline index.html]: Use the article tag instead of div

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @emmanuellekamwa @Guy-Gustave** 
Good job with your first submission for this project :muscle: your page meet most of the requirements :clap: :clap:
There are just a few details to improve before you can go ahead:
- [ ] your README file doesn't have a working link to the Live Demo. You can use this link to generate a Live Demo: https://raw.githack.com/ 
- [ ] remove commented code in your CSS file
- [ ] An important purpose of this project is to make suitable use of the article tag. Please don't overuse div, check your code and follow this simple rule: if a part of your page is a heading followed by paragraphs then use the article tag.
- [ ]  Check HTML markup with this validator https://validator.w3.org/#validate_by_upload the validator lint some issues in your markup.

##### OPTIONAL

- [ ] your footer tag should go after your main tag, not inside

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by BrittanyBlake** [comment]
Pull Request Under Review By Brittany Blake (TSE)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+                <p>Lorem Ipsum Dolor</p>
+            </article>
+            <article class="article">
+                <h2>title of article</h2>
+                <p>Lorem Ipsum Dolor</p>
+            </article>
+          </div>
+        </div>
+        <div class="article-flex">
+            <div class="column1">
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

File: index.html [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+                    Can Data Visualization Improve The Mobile Web Experience?
+                </h2>
+                <p>
+                    by Sansa Stark Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Accusantium voluptates amet rem illum
+                    sequi illo, ex quis eum error ab totam molestias! Pariatur,
+                    qui harum! Ad molestias nulla quisquam unde.
+                </p>
+            </div>
+            <div class="column1">
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

File: index.html [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+                </h2>
+                <p>
+                    by Sansa Stark Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Accusantium voluptates amet rem illum
+                    sequi illo, ex quis eum error ab totam molestias! Pariatur,
+                    qui harum! Ad molestias nulla quisquam unde.
+                </p>
+            </div>
+
+            <div class="column1">
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

File: index.html [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

File: index.html [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+                webinars, discounts and networking — for just 1 coffee a month.
+                Join the family Explore features
+            </p>
+            <div class="footer">
+                <p>Already have an account? Sign in!</p>
+            </div>
+        </aside>
+
+        <article class="article-flex">
+            <div class="column1">
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

File: index.html [hunk 11]
```diff
@@ -0,0 +1,276 @@
...
+                    Can Data Visualization Improve The Mobile Web Experience?
+                </h2>
+                <p>
+                    by Sansa Stark Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Accusantium voluptates amet rem illum
+                    sequi illo, ex quis eum error ab totam molestias! Pariatur,
+                    qui harum! Ad molestias nulla quisquam unde.
+                </p>
+            </div>
+            <div class="column1">
```
Comment (BrittanyBlake) [inline index.html]: Please use the article tag here instead of div as requested by the previous TSE

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @emmanuellekamwa and @Guy-Gustave 👋 
Great job implementing the changes requested by the previous TSE! 🎖️ You have done a wonderful job so far! 👍   However, there are still a few issues that need to be addressed before you can move on 🔽 
****

- [ ] The HTML validator is still showing an issue. Please take care of the following warning:
Screen Shot 2020-06-06 at 10 33 22 AM

- [ ] As pointed out by the previous TSE, an important part of this project is the proper use of the `<article>` tag. There are still a few areas that should be `<article>` tags and not `<div>` tags. You can find more information about when to use an `<article>` tag [here]
(https://www.w3schools.com/tags/tag_article.asp) and [here](https://stackoverflow.com/questions/6947489/html5-appropriate-use-of-article-tag). I have pointed out some of the areas where `<div>` tags should be `<article>` tags but please use the given resources to determine what should be `<article>` tags. 

****

### Optional

- [ ] The elements in your footer should be centered as on the original page:
The original page 👇 
Screen Shot 2020-06-06 at 10 37 33 AM
Your clone 👇 
Screen Shot 2020-06-06 at 10 37 42 AM


****

### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good luck!! 🍀  🤞 

P.S.
If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake).

**Comment by emmanuellekamwa** [comment]
I have implemented every correction on my project as suggested by the TSE. Hope to have a good feedback.

**Comment by BrittanyBlake** [comment]
### Pull Request Under Review By Brittany Blake (TSE)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team 👋 
Great job implementing the requested changes!! 🎖️  It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****
Happy Coding 👩🏾‍💻

**Pull request merged**